src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.spec.ts

```diff
@@ -251,6 +252,17 @@ describe('ServiceFormComponent', () => {
         const ssl_key = fixture.debugElement.query(By.css('#ssl_key'));
         expect(ssl_key).toBeNull();
       });
+
+      it('should test rgw ssl cert', () => {
+        const randExp = new RandExp(
+          /^-----BEGIN RSA PRIVATE KEY-----(\n|\r|\f)((.+)?((\n|\r|\f).+)*)(\n|\r|\f)-----END RSA PRIVATE KEY-----[\n\r\f]-----BEGIN CERTIFICATE-----(\n|\r|\f)((.+)?((\n|\r|\f).+)*)(\n|\r|\f)-----END CERTIFICATE-----[\n\r\f]*$/
```

Comment:
```suggestion
          /^(-----BEGIN .+?-----.+?-----END .+?-----)+*$/
```
We could try to regex the base64, but the expression is not funny and, in the end, the source-of-truth validation should always happen in the back-end.